src/Adapter/Product/Image/Repository/ProductImageRepository.php

```diff
@@ -589,6 +589,15 @@ public function updateMissingCovers(ProductId $productId): void
                 ->setParameter('shopId', (int) $image['id_shop'])
                 ->execute()
             ;
+
+            $this->connection->createQueryBuilder()
+                ->update($this->dbPrefix . 'image')
+                ->set($this->dbPrefix . 'image' . '.cover', ':cover')
```

Comment:
```suggestion
                ->set($this->dbPrefix . 'image.cover', ':cover')
```
can you aslo fix it above in line 584 and 588?

Reply:
It should probably even work like this
```suggestion
                ->set('cover', ':cover')
```
because there ain't no alias and no table joins. Could you check and implement it if it works?